src/adiimu_ros_publisher.cpp

```diff
@@ -49,38 +49,50 @@ void AdiImuRosPublisher::run()
     std::cout << "thread " << this_id << " started...\n";
     RCLCPP_INFO(rclcpp::get_logger("rclcpp"), "startThread: '%d'", this_id);
 
+    auto started = std::chrono::high_resolution_clock::now();
```

Comment:
Do we need to store the time here if we overwrite it later?

Reply:
I used a typedef